### ISSUE TITLE
twisted.web.test.test_woven calls reactor.iterate()

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10912]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1189|
|Type|defect|
|Created|2005-09-19 16:50:33Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1189 1189
type__defect defect
reporter__exarkun exarkun
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__tests tests
time__1127148633000000 1127148633000000
changetime__1142033904000000 1142033904000000
version__ 
owner__radix radix
cc__glyph cc__radix cc__exarkun cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
Hee hee who cares?

Can we get rid of woven already? :)
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
No :-(

web/distrib.py:from twisted.web.woven import page
web/static.py:        from twisted.web.woven import dirlist
web/tap.py:        from twisted.web.woven.flashconduit import FlashConduitFactory
web/widgets.py:from twisted.web.woven import template
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="foom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|foom set status to `closed`|
|-|-|

(In [16223]) Don't use reactor.iterate().
Fixes [#10912](https://github.com/twisted/twisted/issues/10912)
